testing/dom_crawler.rst

```diff
@@ -21,27 +21,29 @@ selects the last one on the page, and then selects its immediate ancestor elemen
 Many other methods are also available:
 
 ``filter('h1.title')``
-    Nodes that match the CSS selector.
+    Selects nodes that match the given CSS selector (which must be supported by
+    Symfony's :doc:`CSS Selector component </components/css_selector>`).
 ``filterXpath('h1')``
-    Nodes that match the XPath expression.
+    Selects nodes matching the given `XPath expression`_.
 ``eq(1)``
-    Node for the specified index.
+    Selects the node at the given index (``0`` is the first node).
 ``first()``
-    First node.
+    Selects the first node (equivalent to ``eq(0)``).
```

Comment:
Using `Selects` in all those descriptions is confusing to me. We have different kinds of methods in this lists:

- methods creating a Crawler containing only a subset of the nodes of the current Crawler instance (`eq` and all its shorcuts, but also `reduce` which actually filters the current instance and `slice`)
- methods returning a new Crawler with a different selection (which *also* includes the `filter` and `filterXpath` methods, despite their name, as they are doing a mix of matching the existing nodes or finding their descendants)

Reply:
You are right, but this doc is in `testing/dom_crawler` (we also have `components/dom_crawler`) so this is probably an internal detail not relevant in this page. 🤔